src/Illuminate/Http/Middleware/ValidatePostSize.php

```diff
@@ -21,7 +21,7 @@ public function handle($request, Closure $next)
         $max = $this->getPostMaxSize();
 
         if ($max > 0 && $request->server('CONTENT_LENGTH') > $max) {
-            throw new PostTooLargeException;
+            throw new PostTooLargeException('The POST data is too large.');
```

Comment:
```suggestion
            throw new PostTooLargeException(__('Content Too Large'));
```

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413